<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilePicker</class>
 <widget class="QWidget" name="TilePicker">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TilePicker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Replace Tile:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="selectedTileLabel">
     <property name="text">
      <string>Tile:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="flowlayout_placeholder_1"/>
   </item>
   <item>
    <widget class="QLabel" name="availableTilesLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;By Tile:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="replacingTileLabel">
     <property name="text">
      <string>Tile:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="flowlayout_placeholder_2"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>